backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -198,37 +203,24 @@ open class SubmissionDatabaseService(
         log.info { "updating processed data" }
         val reader = BufferedReader(InputStreamReader(inputStream))
 
-        val statusByAccessionVersion = mutableMapOf<String, PreprocessingStatus>()
+        val processedAccessionVersions = mutableListOf<String>()
         reader.lineSequence().forEach { line ->
             val submittedProcessedData = try {
                 objectMapper.readValue<SubmittedProcessedData>(line)
             } catch (e: JacksonException) {
                 throw BadRequestException("Failed to deserialize NDJSON line: ${e.message}", e)
             }
 
-            val newStatus = insertProcessedDataWithStatus(submittedProcessedData, organism, pipelineVersion)
-
-            statusByAccessionVersion[submittedProcessedData.displayAccessionVersion()] = newStatus
+            insertProcessedDataWithStatus(submittedProcessedData, organism, pipelineVersion)
+            processedAccessionVersions.add(submittedProcessedData.displayAccessionVersion())
         }
 
-        log.info {
-            var hasErrors = 0
-            var finished = 0
-            for ((_, preprocessingStatus) in statusByAccessionVersion) {
-                when (preprocessingStatus) {
-                    HAS_ERRORS -> hasErrors++
-                    FINISHED -> finished++
-                    IN_PROCESSING -> {}
-                }
-            }
-
-            "Updated $finished sequences to $FINISHED, $hasErrors sequences to $HAS_ERRORS."
-        }
+        log.info("Updated ${processedAccessionVersions.size} sequences to $PROCESSED")
```

Comment:
Slight loss of information here in the logs: we no longer know how many passed with and without errors, but probably not worth complexity to add this back

Reply:
Added it back: [781edae](https://github.com/loculus-project/loculus/pull/3125/commits/781edae3a64c90385a3ae01ed85471bae68a4e30)